lib/openstack_api/openstack_server.py

```diff
@@ -62,15 +63,19 @@ def search_all_servers(
 
         with self._connection_cls(cloud_account) as conn:
             for project in projects:
-                selected_servers.extend(
-                    conn.list_servers(
-                        filters={
-                            "all_tenants": True,
-                            "project_id": project.id,
-                            "limit": 10000,
-                        }
+                try:
+                    selected_servers.extend(
```

Comment:
Can we write a unit test for this, that `list_servers` throwing a HttpException is handled since it's something we discovered in prod?

Also the unit test can be used as the template for a future action too